pkg/BUILD

```diff
@@ -22,16 +22,7 @@ py_library(
         "helpers.py"
     ],
     srcs_version = "PY2AND3",
-    # This points to a fundemental bazel workspace deficiency. What we need is a
-    # way to specify "the workspace that this BUILD or .bzl file comes from".
-    # I do not want to say '@rules_pkg'. That forces my name on everyone who uses
-    # me. '@' seems to be the outer workspace.
-    visibility = [
-        "@//experimental:__pkg__",
-        "@rules_pkg//experimental:__pkg__",
-        "@//tests:__pkg__",
-        "@rules_pkg//tests:__pkg__"
-    ],
+    visibility = ["//visibility:public"],
```

Comment:
Can you add a comment here that this is use at your own risk.
It was private because we didn't want people depending on an internal interface.

Reply:
Done.